### PR TITLE
Fix/click see all search

### PR DESCRIPTION
**Pull request by codomposer**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #<10650>
https://github.com/kodadot/nft-gallery/issues/10650

## Screenshot 📸
image
image
image

### CODE REVIEW

**Review by Jarsen136 — CHANGES_REQUESTED** [review]

File: components/search/SearchSuggestion.vue [hunk 1]
```diff
@@ -416,28 +403,27 @@ const selectedItemListMap = computed(() => ({
...
 const { $consola } = useNuxtApp()
 
-const updateSearchUrl = () => {
+const updateSearchUrl = (path: string) => {
   const { name } = props
   if (name) {
     router
       .replace({
-        path: String(route.path),
+        path: `${path}`,
```
Comment (hassnian) [inline components/search/SearchSuggestion.vue:413]:
```suggestion
        path,
```

File: components/search/SearchSuggestion.vue [hunk 2]
```diff
@@ -390,10 +378,9 @@ const nftSuggestion = computed(() =>
 
 const loadMoreItemClassName = computed(
   () =>
-    `link-item${
-      selectedIndex.value === totalItemsAtCurrentTab.value
-        ? ' selected-item'
-        : ''
+    `link-item${selectedIndex.value === totalItemsAtCurrentTab.value
+      ? ' selected-item'
```
Comment (hassnian) [inline components/search/SearchSuggestion.vue:382]:
```suggestion
      ? 'selected-item'
```

**Comment by exezbcz** [comment]
it does not redirect to the explorer when first hitting the enter - its the first search i made on the page 
image



looks like edge case, could you please investigate?

**Comment by codomposer** [comment]
The functionality is working correctly. However, before getting all JavaScript and asset files, it does not function as expected. Unfortunately, there is no workaround for this issue.

**Comment by exezbcz** [comment]
second review? @kodadot/internal-dev

**Comment by codomposer** [comment]
https://github.com/kodadot/nft-gallery/issues/10650

**Comment by vikiival** [comment]
Thank you for your contribution!
Seems that issue was not merged for one month, therefore I am closing that as stale.
Please pickup other p1/p2 issue ^-^.

**Pull request closed without merging**